helm/reana/values.yaml

```diff
@@ -57,31 +57,31 @@ db_env_config:
 components:
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server
+    image: reanahub/reana-server:0.6.0-35-gafd0a7f
```

Comment:
Hmm, this would break local production-like build workflows:

```console
$ DEMO=r-d-r-roofit EXCLUDE_COMPONENTS=r-ui,r-m-broker make ci
```

as these would not use `latest` anymore, but the released versions, isn't it?

If so, we really need those several configuration files, as I mentioned in
https://github.com/reanahub/reana/pull/289#pullrequestreview-394489878, for example:

```
configurations/minikube-dev-latest.yaml
configurations/minikube-prod-latest.yaml
configurations/minikube-prod-released.yaml
...
```

Or else some `sed` tricks in `Makefile` to bring back the `latest` for these cases.

Reply:
It shouldn't break the local production-like build as the [`values.yaml` path](https://github.com/reanahub/reana/pull/289/files#diff-b67911656ef5d18c4ae36cb6741b7965R31) depends on whether the BUILD is a release or development.
```console
$ BUILD_TYPE=release make ci # will run CI pointing to release tags
$ make ci # implicitly being BUILD_TYPE=development, will always point to latest
```

Maybe what could be improved is the naming? We have `BUILD_TYPE` being one of `['release', 'development']`, that's where the confusion comes I guess? Something like `['release', 'local']` or any other suggestion you have.

Reply:
`BUILD_TYPE` is equal to `development` by default, but if you launch the `DEMO=r-d-r-roofit EXCLUDE_COMPONENTS=r-ui,r-m-broker make ci` command I mentioned above, the deployment procedure tries to download tagged images on my system instead of using `latest`... Have you tried to run it in your "reana" production-like environment?

Ah, I see where the error is:

```console
$  make deploy -n
bash: line 0: test: development: integer expression expected
...
helm install reana helm/reana  -fhelm/reana/values.yaml --wait && \
```

A wrong `test` statement when setting `VALUES_YAML_PATH`.  

Here's the fix:

```diff
diff --git a/Makefile b/Makefile
index c230276..909bab4 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ DEBUG := $(shell grep -q 'debug.enabled=true' <(echo ${CLUSTER_FLAGS}) && echo 1
 PWD := $(shell pwd)
 REANA_CODE_DIR=$(shell cd ..; pwd)
 TRUNC_INSTANCE_NAME=$(shell echo ${INSTANCE_NAME} | head -c 10 | xargs echo)
-VALUES_YAML_PATH := $(shell test ${BUILD_TYPE} -eq development && echo helm/reana-dev/values.yaml || echo helm/reana/values.yaml)
+VALUES_YAML_PATH := $(shell test ${BUILD_TYPE} = "development" && echo helm/reana-dev/values.yaml || echo helm/reana/values.yaml)

 all: help
```